src/SFA.DAS.AANHub.Api.UnitTests/Controllers/CalendarEventsControllerTests/CalendarEventsControllerGetTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using FluentValidation.Results;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.AANHub.Api.Controllers;
+using SFA.DAS.AANHub.Application.CalendarEvents.Queries.GetCalendarEvent;
+using SFA.DAS.AANHub.Application.Mediatr.Common;
+using SFA.DAS.AANHub.Application.Mediatr.Responses;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.AANHub.Api.UnitTests.Controllers.CalendarEventsControllerTests;
+
+public class CalendarEventsControllerGetTests
+{
+    [Test]
+    [MoqAutoData]
+    public async Task GetCalendarEventById_InvokesQueryHandler(
+       [Frozen] Mock<IMediator> mediatorMock,
+       [Greedy] CalendarEventsController sut,
+       Guid calendarEventId,
+       Guid requestedByMemberId)
+    {
+        await sut.Get(calendarEventId, requestedByMemberId);
+
+        mediatorMock.Verify(m => m.Send(It.Is<GetCalendarEventByIdQuery>(q => q.CalendarEventId == calendarEventId), It.IsAny<CancellationToken>()));
```

Comment:
can add requestedMemberId check here as well.

Reply:
Done, and I've also tried to do some housekeeping by shortening the test names and updating NUnit/AutoFixture packages - please re-review and approve if happy